### PR TITLE
 - cleaned up code and messages.

### PR DESCRIPTION
**Pull request by guidotijskens**
- cleaned up code and messages, removed and modified console and debug messages, reducing possible confusion.
 - added oci-image-migrate-upload code: a separate script to upload a (verified and modified) on premise image to object storage in the OCI only; the oci-image-migrate script still has the upload part, but by separting thig, the image can be uploaded more than once, to different object storage, without the need to go through the verification and modification.
 - modified code so upload and import can be run as non-root user; the upload and import do not need the root user, the verification/modification does due to the mount and package install operations.
 - updated spec file so oci-utils-migrate...el8.noarch.rpm is not build, is not yet supported; due to the absence of the nbd kernel module in OL8/RH8, the oci-image-migrate script will not run, building the el8 rpm would cause confusion and bug reports. This is is being worked on.
- removed the tests from the rpm, they are not yet complete.
.
the changes have been tested with real life images.

### CODE REVIEW

File: man/man1/oci-image-migrate-import.1 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .\" Process this file with
-.\" groff -man -Tascii oci-image-migrate.1
+.\" groff -man -Tascii oci-image-migrate-import.1
 .\"
 .\" Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
 .\"
```
Comment (wbora) [inline man/man1/oci-image-migrate-import.1]: You need to include the full license piece into the file, ie, these lines are missing:

Licensed under the Universal Permissive License v 1.0 as shown at
 http://oss.oracle.com/licenses/upl.

File: man/man1/oci-image-migrate-upload.1 [hunk 2]
```diff
@@ -0,0 +1,95 @@
+.\" Process this file with
+.\" groff -man -Tascii oci-image-migrate-upload.1
+.\"
+.\" Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (wbora) [inline man/man1/oci-image-migrate-upload.1]: Same comment as above the full license file header block need to be included

File: man/man1/oci-image-migrate.1 [hunk 3]
```diff
@@ -54,7 +54,7 @@ The full path name of the on-premise custom image. This option is mandatory.
 .IP "-b|--bucket"
```
Comment (wbora) [inline man/man1/oci-image-migrate.1]: This is a another FILE comment, not for this line, the top license block need to be completed

**Comment by guidotijskens** [comment]
updated upl banners, added oci-image-migrate-upload.md

File: man/man1/oci-image-migrate.1 [hunk 4]
```diff
@@ -3,7 +3,6 @@
 .\"
 .\" Copyright (c) 2017, 2019 Oracle and/or its affiliates. All rights reserved.
 .\"
```
Comment (wbora) [inline man/man1/oci-image-migrate.1]: Copyright message still not complete here

Comment (guidotijskens) [reply man/man1/oci-image-migrate.1]: fixed

**Comment by guidotijskens** [comment]
fixed the permission denied errors created by result (and log?) messages when not root user.
please have  a look/
tx

**Comment by ejannett** [comment]
> fixed the permission denied errors created by result (and log?) messages when not root user.
> please have a look/
> tx

Hi,

What's in the last changes refere to logging ?

**Pull request merged**